src/components/Home/index.tsx

```diff
@@ -88,6 +88,22 @@ const Overview: React.FC<{
           linkLabel="View Logs"
           testId="emulator-info-functions"
         />
+        <EmulatorCard
+          name="Hosting Emulator"
+          icon={<FunctionsIcon theme="secondary" />}
+          config={config.hosting}
+          testId="emulator-info-hosting"
+          linkToExternal={
+            config.hosting && `http://${config.hosting?.hostAndPort}/`
```

Comment:
```suggestion
            config.hosting && `http://${config.hosting.hostAndPort}/`
```

I'm pretty sure TypeScript will narrow this down, no? Otherwise, use `config.hosting!.hostAndPort`.

Reply:
Yeah, that works. thanks